pkg/network/node/runtime.go

```diff
@@ -85,6 +87,17 @@ func (node *OsdnNode) getPodSandboxes() (map[string]*kruntimeapi.PodSandbox, err
 
 	podSandboxMap := make(map[string]*kruntimeapi.PodSandbox)
 	for _, sandbox := range podSandboxList {
+		status, err := runtimeService.PodSandboxStatus(sandbox.Id)
```

Comment:
PodSandboxStatus() might end up calling the CNI plugin (eg, openshift-sdn again) and if the SDN isn't running yet could return an error. Do we really want to ignore the sandbox in that case?

Reply:
> PodSandboxStatus() might end up calling the CNI plugin again

No, `PodSandboxStatus()` does not mean "do a CNI `CHECK`". It just gets the runtime's current data for the pod.